rkovalenko/ReverseBTree.cs

```diff
@@ -0,0 +1,33 @@
+/**
+ * Definition for a binary tree node.
+ * public class TreeNode {
+ *     public int val;
+ *     public TreeNode left;
+ *     public TreeNode right;
+ *     public TreeNode(int val=0, TreeNode left=null, TreeNode right=null) {
+ *         this.val = val;
+ *         this.left = left;
+ *         this.right = right;
+ *     }
+ * }
+ */
+public class Solution {
+    public TreeNode InvertTree(TreeNode root) {
+        return RevertNode(root);
+    }
+    
+    public TreeNode RevertNode(TreeNode node)
+    {
+        if(node is not null)
```

Comment:
should have inverted the if statement 🤔

Reply:
It's more aesthetically pleasing that way because it's shorter and no premature exit, but that's just personal taste.